roles/dns/templates/named.service.j2

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Requires=docker.service
+After=docker.service
+
+[Service]
+Type=simple
+TimeoutStartSec=5m
+ExecStartPre=-/usr/bin/docker kill bind
+ExecStartPre=-/usr/bin/docker rm bind
+ExecStart=/usr/bin/docker run --name bind -d -p 53:53/udp -v /var/log:/var/log -v /etc/named.conf:/etc/named.conf -v /var/named:/var/named:z bind
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
We've started using changing these to use:
```
WantedBy=docker.service
```
This ensures that the container is started regardless of how docker is started/stopped/restarted.